collector/lib/NRadix.h

```diff
@@ -80,7 +80,6 @@ class NRadixTree {
       auto inserted = this->Insert(network);
       if (!inserted) {
         CLOG(ERROR) << "Failed to insert CIDR " << network << " in network tree";
-        delete root_;
```

Comment:
I agree that there is an alternative which consists in deleting `root_` *and* setting it to 0.

Since the only reason for insert to fail is currently to have a duplicated network, I chose to just ignore such duplicate entry.